client/my-sites/email/titan-set-up-mailbox/titan-set-up-mailbox-form.tsx

```diff
@@ -91,10 +86,12 @@ const useHandleCompleteSetup = (
 		try {
 			await mutateAsync( mailbox.getAsFlatObject() as TitanMailboxFields );
 
-			goToThankYouPage(
-				selectedSite?.slug as string,
-				selectedDomainName,
-				mailbox.formFields.mailbox.value
+			// There is some bug at the backend - it's not created instantly, so as temporary fix I added 5000 delay
+			await new Promise( ( resolve ) => setTimeout( resolve, 5000 ) );
+
+			page( emailManagementEdit( selectedSite?.slug as string, selectedDomainName ) );
+			dispatch(
+				successNotice( translate( 'Your email is now ready to use!' ), { duration: 5000 } )
```

Comment:
```suggestion
				successNotice( translate( 'Your email is now ready for use!' ), { duration: 5000 } )
```
Sorry I think this copy came from me 😓 Reading it again, I'm not quite sure if that's the right grammar. Do you mind checking the existing Congrats pages to see if there might be similar copy that we can reference, whether it makes sense to go with `ready to use` or `ready for use`?

Reply:
ChatGPT says that `for use` is more formal, and `to use` is more informal. But both are ok.
As i see, typically we have approach with notifications as `... successfully` / `... sent` / `... enabled`.

Reply:
Ok. As long as it isn't grammatically incorrect, let's keep it as-is and not delay translation!